README.md

```diff
@@ -1,3 +1,17 @@
 # atd-ped-bike-crash
 
 A repo for work related to the Austin Vision Zero team's project to analyze pedestrian and bicycle crashes. 
+
+## Trail Counters Data
```

Comment:
I agree that this feels out of place in this repo, but I don't have a lot of context for this repo yet since it's new. Maybe since this concerns an airflow dag, you could make a new folder to hold one-offs and spikes in the airflow repo? Making a new repo too is great if this is going to be an on-going project. I'm curious what more senior team members think, for sure.

Reply:
I kinda think we should've put most of the things in this repo in https://github.com/cityofaustin/atd-vz-data if the output is for Vision Zero for mobility safety.

Reply:
Trail counter data was mentioned as a data source for the analysis VZ is doing for ped/bike crashes. I'm leaning towards putting this in its own repo or moving this to a folder in this repo called `data` or something?